scripts/zbtestcrypt

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env python
+
+#  zbtestcrypt - test encryption by decrypting and re-encrypting
```

Comment:
Let's merge this as a tool for now, but really this I think would be better as a pytest test case, with a PCAP file included that it works off of to ensure everything passes.